### DISCUSSION TITLE
For Quick Start

### DISCUSSION THREAD

**Discussion by touirssa** [simdjson/simdjson#1997] [Q&A]
I have got the following error at compiling stage:
$ c++ -o quickstart quickstart.cpp simdjson.cpp
simdjson.cpp: In member function ‘simdjson::error_code simdjson::fallback::{anonymous}::stage2::tape_builder::visit_root_true_atom(simdjson::fallback::{anonymous}::stage2::json_iterator&, const uint8_t*)’:
simdjson.cpp:6672:10: error: ‘idx’ was not declared in this scope
   return idx < lenminusstep;
          ^~~
simdjson.cpp:6672:10: note: suggested alternative: ‘index’
   return idx < lenminusstep;
          ^~~
          index
simdjson.cpp:6672:16: error: ‘lenminusstep’ was not declared in this scope
   return idx < lenminusstep;
                ^~~~~~~~~~~~
simdjson.cpp: In member function ‘bool simdjson::icelake::{anonymous}::buf_block_reader<STEP_SIZE>::has_full_block() const’:
simdjson.cpp:7222:1: warning: no return statement in function returning non-void [-Wreturn-type]
 }
 ^

**Comment by lemire** [comment]
You are getting an error at line 6672 from the singleheader simdjson.cpp, correct? That line has the following content:

https://github.com/simdjson/simdjson/blob/e9f0f1bb9baa7fc2ba1ead6191e5d3f9e924d64f/singleheader/simdjson.cpp#L6672

<img width="1235" alt="Screenshot 2023-05-03 at 10 10 48 PM" src="https://user-images.githubusercontent.com/391987/236096097-4f8ab177-6d33-4f26-9079-9e30c0c51b8c.png">

**Comment by lemire** [comment]
I have also checked from our last release and line 6672 from simdjson.cpp does not match the content you are reporting.

Are you certain you have the correct file?

**Reply by touirssa** [reply]
Thanks Lemire, I have picked up the bad  release
